templates/kubernetes/1/rancher-compose.yml

```diff
@@ -0,0 +1,74 @@
+.catalog:
+  name: "Kubernetes"
+  version: "v1.2.4-rancher4"
+  description: "Rancher Kubernetes service"
+  minimum_rancher_version: v1.1.0
```

Comment:
for rc build, this template is not being returned. Should we put an rc version here? @prachidamle just to confirm that the rc template will be returned to the v1.1.0 release